README.md

```diff
@@ -1,4 +1,6 @@
+<<<<<<< Updated upstream
```

Comment:
Seems like this shouldn't be here

Reply:
I feel like a training resource to train reviewing